examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/with-scaffolder.py

```diff
@@ -0,0 +1,79 @@
+import os
+import subprocess
+from collections.abc import Sequence
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Annotated, Any
+
+from dagster_components import (
+    AssetSpecSchema,
+    Component,
+    ComponentLoadContext,
+    FieldResolver,
+    ResolutionContext,
+    ResolvableSchema,
+    registered_component_type,
+)
+from dagster_components.core.component_scaffolder import (
+    ComponentScaffolder,
+    ComponentScaffoldRequest,
+)
+from dagster_components.scaffold import scaffold_component_yaml
+
+import dagster as dg
+
+
+class ShellScriptSchema(ResolvableSchema):
+    script_path: str
+    asset_specs: Sequence[AssetSpecSchema]
+
+
+def resolve_asset_specs(
```

Comment:
This shouldn't be necessary since `AssetSpecSchema` is `ResolvableSchema[AssetSpec]`

Reply:
Good call. I think we still want an example of this annotated custom resolution logic, so pushed it to a new advanced section later in the page.